draft-ietf-tls-esni.md

```diff
@@ -534,9 +537,10 @@ nonce
 : A random 16-octet value to be echoed by the server in the
 "encrypted_server_name" extension.
 
-sni
-: The true SNI, that is, the ServerNameList that would have been sent in the
-plaintext "server_name" extension.
+dns_name
+: The true SNI DNS name, that is, the HostName value that would have been sent in the
+plaintext "server_name" extension. (NameType values other than "host_name" are 
+unsupported since SNI extensibility failed {{SNIExtensibilityFailed}}).
```

Comment:
I don't know if this is the best place to signal this, but I approve anyway.

Reply:
I figured it's best to be honest and up front. :-)